docs/design/proposals/encrypted-with-keyprotect.md

```diff
@@ -16,7 +16,7 @@ service from the CSI driver and to make use of the encryption operations:
 The Key Protect/HPCS connection URL.
 
 * IBM_KP_TOKEN_URL
-The Token Authenticaltion URL of KeyProtect/HPCS service.
```

Comment:
better to name the file as `encryption-with-keyprotect`?

Reply:
yeah, kept that way.